end2end-test-examples/gcs/src/main/java/io/grpc/gcs/TestMain.java

```diff
@@ -29,10 +22,6 @@ public static void main(String[] args) throws Exception {
     if (a.conscrypt) {
       Security.insertProviderAt(Conscrypt.newProvider(), 1);
     }
-    if (a.autoWindow) {
```

Comment:
Can we keep this option in an opposite way? gRPC 1.30 enables this by default so let's change this like; if autoWindow=false (by default true), disable flow control. (using NettyChannelBuilder.flowControlWindow)

Reply:
Or chaging the `window` option to do this would be an alternative. When user specify window option, it configured the window size by given value and disables the flow control.

Reply:
The current logic is, if `window` option is set to any number larger than 0, then NettyChannelBuilder.flowControlWindow is used to set to the given number, does that already serves the purpose of disabling BDP?

Reply:
Oh I see then it'd work! Thanks!